spec/workers/mail_notification_jobs_spec.rb

```diff
@@ -64,9 +64,18 @@
         expect(mail).not_to be_present
       end
 
-      it 'deos not raise an error but fails silently' do
+      it 'does not raise an error but fails silently' do
         expect{job.perform}.not_to raise_error
       end
+
+      context 'raising exceptions' do
+        before { MailNotificationJob.raise_exceptions = true }
+        after { MailNotificationJob.raise_exceptions = false }
```

Comment:
This configuration is supposed to be done in the code in some initializer.
But if you insists on the configuration via ENV variables, I will add it probably into [configuration.rb](https://github.com/barhot/openproject/blob/dev/lib/open_project/configuration.rb)